### PR TITLE
Remove duplicate volumes

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: salt/journal/config/srv-journal-docker-compose.yml [hunk 1]
```diff
@@ -16,8 +16,6 @@ services:
         env_file:
             - ./containers.env
         restart: always
-        volumes:
-            - ./parameters.yml:/srv/journal/app/config/parameters.yml
```
Comment (giorgiosironi) [inline salt/journal/config/srv-journal-docker-compose.yml]: if I understand correctly, in YAML this `volumes` key completely override the previous `volumes` key

**Comment by giorgiosironi** [comment]
Permission problem?
```
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:     [0;31m      ID: journal-cache-warmup[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:     [0;31mFunction: cmd.run[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:     [0;31m    Name: docker-compose exec -T fpm bin/console cache:warmup[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:     [0;31m  Result: False[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:     [0;31m Comment: Command "docker-compose exec -T fpm bin/console cache:warmup" run[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:     [0;31m Started: 10:42:30.526596[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:     [0;31mDuration: 7913.151 ms[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out: [0;31m     Changes:   
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:               [0;36m----------[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:               [0;36mpid[0;0m:
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:                   [0;1;33m9906[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:               [0;36mretcode[0;0m:
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:                   [0;1;33m1[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:               [0;36mstderr[0;0m:
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:                   [0;32m[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:                   [0;32mIn Kernel.php line 762:[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:                   [0;32m                                                                        [0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:                   [0;32m  Unable to write in the logs directory (/srv/journal/src/../var/logs)[0;0m
[2019-08-29T10:42:46.136Z] [3.90.210.35] out:               [0;36mstdout[0;0m:[0;0m
```

**Comment by giorgiosironi** [comment]
https://github.com/elifesciences/journal-formula/blob/master/salt/journal/init.sls#L80-L97 intends for `var/logs/` to be automatically created with `www-data` group ownership. Will reproduce manually to find out why the container can't write.

**Comment by giorgiosironi** [comment]
Notes follow.

The `journal_fpm_1` container is unhealthy for the same reason in the manually created stack. It is running as `www-data`.

The container and the host both see:
```
drwxr-sr-x 2 root     root     4096 Aug 29 11:03 logs
```
so the group ownership is not as intended as it should be `www-data` due to the sticky group bit:
```
elife@pr-99-manual--journal:/srv/journal$ ls -ld var/
drwxrwsr-x 3 www-data www-data 4096 Aug 29 11:03 var/
```
Since it's `docker-compose` that creates this folder, it likely ignores it?
Besides, `logs` is not writable by its group so it wouldn't work anyway.

Going to create `var/logs/` manually with the right ownership and permissions.

**Comment by giorgiosironi** [comment]
c3dc4eb0b52abc745798dc4e8a58ec8dd81b76d8 correctly fixes `var/logs/`; it will also skip setting the same behavior on `var/` but as far as I can tell it's unused as for example `var/cache/` is only present inside the container.

Ready to go :+1:

File: salt/journal/init.sls [hunk 2]
```diff
@@ -77,13 +77,14 @@ assets-nginx-configuration:
 # files and directories must be readable and writable by both elife and www-data
```
Comment (giorgiosironi) [inline salt/journal/init.sls]: will remove these outdated comments later

**Comment by thewilkybarkid** [comment]
Problematic to test locally as Redis isn't initialized properly and the `dev` environment logs to `stderr` anyway.

**Pull request merged**